package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@actions/http-client",
-  "version": "1.0.7",
+  "version": "1.0.8",
```

Comment:
only a patch bump to ensure as many folks as possible just get the fix.  I will also post a security advisory